modules/javascript/pages/quickstart.adoc

```diff
@@ -36,22 +36,11 @@ image::ROOT:manage-and-store-data-locally.svg[,250]
 
 // include::ionic.adoc[tag="summary"]
 
-* https://ionic.io/integrations/couchbase-lite[Couchbase{nbsp}Lite{nbsp}Integration]
-
-* https://ionic.io/docs/couchbase-lite[Documentation]
-
-* https://ionic.io/docs/couchbase-lite/tutorials/hotel-search[Tutorial]
-
 * https://capacitorjs.com/docs/plugins[Capacitor]
 
-[.column]
-====== {empty}
-[.content]
-.xref:javascript:cordova.adoc[Cordova]
+* https://cbl-ionic.dev[Plugin{nbsp}Documentation]
```

Comment:
Not sure if you need the {nbsp} here - I realise it's copied from the existing links so not a biggie, just a possible opportunity for cleanup 🤷

Reply:
Tested without and seems fine 🤷🏼 , removed!